scripts/Deploy-CustomScriptsExtension.ps1

```diff
@@ -58,72 +58,37 @@ Param(
         [Parameter(Mandatory=$true)][string]$AadTenantId,
         [Parameter(Mandatory=$true)][string]$TemplateFile,
         [Parameter(Mandatory=$true)][string]$TemplateParameterFile,
+
+        [Parameter(Mandatory=$true)][string]$ClusterAdmininistratorEmailAddress,
+        [Parameter(Mandatory=$true)][ValidateSet("oms","datadog")][string]$AgentType,
+        [Parameter(Mandatory=$true)][string]$PrimaryKey,
+        [Parameter(Mandatory=$false)][string]$OmsWorkspaceId="-",
+        [Parameter(Mandatory=$true)][string]$DeploymentVersionId,
+        [Parameter(Mandatory=$false)][string]$OxaToolsGithubAccountName="Microsoft",
+        [Parameter(Mandatory=$false)][string]$OxaToolsGithubProjectName="oxa-tools",
+        [Parameter(Mandatory=$false)][string]$OxaToolsGithubBranch="oxa/master.fic.eltonc",
```

Comment:
Not quite. Those are default parameters which can be overwritten. Should be updated anway.